.githooks/pre-commit

```diff
@@ -1,4 +1,23 @@
 #!/usr/bin/env bash
 
-cargo make check-rs-format && \
-cargo make check-ts-format
+# Run check-rs-format if Rust source files were modified.
+git status | egrep "*.rs"
+if [ $? -eq 0 ]; then
+    cargo make check-rs-format
+    ret=$?
+    if [ $ret -ne 0 ]; then
```

Comment:
:wrench: I think you can avoid this if you do `set -e` at the beginning of the script.

Reply:
We can't use `set -e` here because `grep` failure is part of the normal code path.